### PR TITLE
TP: Persist hidden (by default) column filters on reload

### LINKED ISSUE #5294: Column filter does not persist when multiple columns share a valueFormatter
## I'm submitting a ...
-  bug report

## Traffic Control components affected ...
-  Traffic Portal

## Current behavior:
Navigate to https://tp.domain.tld/#!/servers and set a filter on the Reval Pending column like such:

image

check local storage and you should see the following entry:

```
servers_table_filters: {"revalPending":{"filterType":"text","type":"contains","filter":"true"}}
```

Refresh your browser. The filter is lost.

## Expected behavior:
Refresh your browser. The filter persists.

## Minimal reproduction of the problem with instructions:
See above

## Anything else:
The same seems to happen with the Status Last Updated and Last Updated columns. What do these things have in common? The columns share a valueFormatter (UpdateCellRenderer and dateCellFormatter respectively) with another column. That is my guess.

### PR DESCRIPTION
**Pull request by shamrickus**
## What does this PR (Pull Request) do?

- [x] This PR fixes #5294 and #5295

The issue was that the column filters (on hidden by default columns) are loaded from local storage before the onColumnVisible method was called, resulting in the loaded filters being wiped out.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Run TP and add a filter to a column that is not visible by default. Refresh the page and confirm that the column is still visible.

## If this is a bug fix, what versions of Traffic Control are affected?

- master
- 5.0.x (RC1)

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_portal/app/src/common/modules/table/deliveryServiceRequests/TableDeliveryServiceRequestsController.js [hunk 1]
```diff
@@ -218,6 +218,75 @@ var TableDeliveryServicesRequestsController = function (tableName, dsRequests, $
 				}
 			}
 		},
+        onFirstDataRendered: function() {
```
Comment (mitchell852) [inline traffic_portal/app/src/common/modules/table/deliveryServiceRequests/TableDeliveryServiceRequestsController.js:221]: minor format issue

**Comment by mitchell852** [comment]
@shamrickus - sorry, one more table was converted to ag-grid between now and the time this PR was submitted - TableChangeLogsController. Want to update that too?

**Comment by shamrickus** [comment]
Sure

**Comment by mitchell852** [comment]
also, @shamrickus want to fix #5295 while you're at it in this PR or create another PR? your choice. super easy one.

**Review by mitchell852 — APPROVED** [review]
LGTM!

**Comment by mitchell852** [comment]
dang, conflict

**Comment by shamrickus** [comment]
Conflicts are resolved

**Pull request merged**